mqtt_exporter/main.py

```diff
@@ -79,29 +86,51 @@ def _normalize_prometheus_metric_name(prom_metric_name):
     return prom_metric_name
 
 
-def _create_prometheus_metric(prom_metric_name):
+def _normalize_prometheus_metric_label_name(prom_metric_label_name):
+    """Transform an invalid prometheus metric to a valid one.
+
+    https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels
+    """
+    if prom_metric_label_name.startswith("__"):
```

Comment:
I think the test should be after the line 104, to avoid double underscores if it is starting with an invalid char.

Reply:
Good point. Although, that means we can't just return on a "valid" name anymore, because then we would allow double underscores.